### ISSUE TITLE
Bug - Bforartists API "bpy.ops.3dview" and issue with Blender API operators starting with numbers

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#1643]
# Issue
![image](https://user-images.githubusercontent.com/35357409/85476235-7499d700-b57d-11ea-9e58-fbe68c3df6c2.png)
Right now, some addons are pulling errors because operators can't really be called when they start with a number - causing a typical syntax error. Bforartists seems to have a ops that starts with a number, which is registered but shouldn't exist and these cannot start with numbers. 

This is the **3dview** one. Blender doesn't have this. 

To test the issue, run the following commands in the script editor:

Console:
`dir(bpy.ops)`

You should see the list like this in Blender, to compare. 
![image](https://user-images.githubusercontent.com/35357409/85476721-829c2780-b57e-11ea-847e-c2ca846008b2.png)

Now run the following:

Console:
`bpy.ops.3dview`

![image](https://user-images.githubusercontent.com/35357409/85476546-2a652580-b57e-11ea-8611-e6331872c9ea.png)
You should get this error. 

# Task
Find why this is used, and.. check and see if it should be more like Blender by not starting with a number for addon and api compatibility.

### Notes
This was troubleshooted by the kind developers of the Blender Visual Scripting addon, which makes addons through visual scripting and nodes. 

**Version:** devbuild-21-06-2020

**Comment by ReinerBforartists** [comment]
Odd, for sure. We need to remove 3dview it seems.

**Comment by ReinerBforartists** [comment]
I am still not sure what you mean with a number. I will check the api now. But could you please give me an example what addon throws an error? If possible a free one?

**Comment by ReinerBforartists** [comment]
This is more than one bug at once. Please just make one report for one issue.

**Comment by ReinerBforartists** [comment]
Fixed the 3Dview issue

**Comment by Draise14** [comment]
As far as I know, I think I'm only reffering to one bug with that particular entry you've fixed already, closing this essentially. 

I'll give it a test now. 

The extra note about the addon issues is just an observation of how I discovered the nuance. I have only found one other addon that tries to use another operator with a number and that too causes itself issues, the **Align Tools Addon**, with it's own that says **bpy.ops.2xyz**. We don't have to fix that.
